blockly-core/test.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+cd tests
+
+# Make sure phantomjs 2+ is installed
+phantomjsversion=$(node_modules/phantomjs/bin/phantomjs --version)
+if [ $phantomjsversion != "2.0.0" ]; then
+  npm install
```

Comment:
If we remove the automated npm install here, I must have to update some other script to make sure our CI environments npm install as needed, right?  Do you know where that is?